backend/api/migrations/0043_ohshownevent_sight_see_date_time.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.13 on 2022-04-24 08:33
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0042_load_initial_data'),
+    ]
+
+    operations = [
```

Comment:
@EagleC0318 @yanghaochang104 codes like this are generated by `manage.py makemigrates`, which is a part of the command line feature provided by Django framework.

Reply:
Cool, so all there migration files are generated by the `manage.py makemigrates`? Do you need to put some different args for them?

Reply:
Yes (generated by `manage.py makemigrates`.
No, I don't need to put extra args.